unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MemoryManager/MemoryManager.cs

```diff
@@ -7,20 +7,20 @@ namespace DCL
 {
     public class MemoryManager : IMemoryManager
     {
-        private const uint MAX_USED_MEMORY = 1300 * 1024 * 1024;
-        private const float TIME_FOR_NEW_MEMORY_CHECK = 1.0f;
+        private const long MAX_USED_MEMORY = 1300 * 1024 * 1024; // 1.3GB
+        private const float TIME_FOR_NEW_MEMORY_CHECK = 1.0f; // Each second
 
         private Coroutine autoCleanupCoroutine;
 
-        private uint memoryThresholdForCleanup = 0;
+        private long memoryThresholdForCleanup = 0;
```

Comment:
**Suggestion**: I notice these 2 variables are being set with the value of `MAX_USED_MEMORY` and `TIME_FOR_NEW_MEMORY_CHECK` constants in the class constructor and they are never change anymore, so... couldn't we remove the variables and simply use the constants directly?